pandas/core/sparse/scipy_sparse.py

```diff
@@ -116,14 +116,19 @@ def _sparse_series_to_coo(ss, row_levels=(0, ), column_levels=(1, ),
     return sparse_matrix, rows, columns
 
 
-def _coo_to_sparse_series(A, dense_index=False):
+def _coo_to_sparse_series(A, dense_index=False, sparse_series=True):
     """
     Convert a scipy.sparse.coo_matrix to a SparseSeries.
```

Comment:
Done. I'm not really sure on two things

1. The type for A is `'scipy.sparse.coo.coo_matrix'`, but we can't import sparse.
2. The return type is `Union[Series, SparseSeries]` but importing SparseSeries would cause a circular import

so I left types off for those.

Reply:
1. can't you just use the string? (I think that works)
2. same use the string

Reply:
Can you? Are these types actually checked in our CI? I'd rather not introduce invalid types.

Reply:
yes they *should* be